agrotoxico/views/agrotoxico.py

```diff
@@ -1,7 +1,15 @@
-from agrotoxico.serializers.agrotoxico import AgrotoxicoSerializer
+from agrotoxico.models import Agrotoxico
+from agrotoxico.serializers.agrotoxico import AgrotoxicoCreateSerializer, AgrotoxicoListSerializer
 
-from rest_framework.generics import CreateAPIView
+from rest_framework.generics import ListCreateAPIView
 
 
-class AgrotoxicoAPIView(CreateAPIView):
-    serializer_class = AgrotoxicoSerializer
+class AgrotoxicoAPIView(ListCreateAPIView):
+    queryset = Agrotoxico.objects.all()
+    write_serializer_class = AgrotoxicoCreateSerializer
+    read_serializer_class = AgrotoxicoListSerializer
+
+    def get_serializer_class(self):
+        if self.request.method == "GET":
```

Comment:
```suggestion
        if self.request.method.lower() == "get":
```
Tomar cuidado para caso não venha tudo maiúsculo ou minúsculo